autogluon/utils/tabular/ml/models/tabular_nn/tabular_nn_model.py

```diff
@@ -514,55 +527,46 @@ def process_train_data(self, df, labels):
         self.feature_type_map = self._get_feature_type_map() # OrderedDict of feature-name -> feature_type string (options: 'vector', 'embed', 'language')
         # print(self.feature_type_map)
         return TabularNNDataset(df, self.feature_arraycol_map, self.feature_type_map,
-                                batch_size=self.params['batch_size'], num_dataloading_workers=self.params['num_dataloading_workers'],
+                                batch_size=batch_size, num_dataloading_workers=num_dataloading_workers,
                                 problem_type=self.problem_type, labels=labels, is_test=False)
 
-    def setup_trainer(self, train_dataset=None):
+    def setup_trainer(self, params, train_dataset=None):
```

Comment:
should remove "loss" from docstring of this function now to avoid confusion

Reply:
Agreed, will update